### PR TITLE
Improve "Cannot find .abapgit.xml" situation

### LINKED ISSUE #4174: Error while creating repository, "Cannot find .abapgit.xml - Is this an abapGit repo?"
Lost of non abapgit related files existed in the repository, the creation of the repo should not fail, its okay to show the error message in the repo screen

image

image

### PR DESCRIPTION
**Pull request by mbtools**
Skip check during creating of repo and increase file count.

Closes #4174

### CODE REVIEW

**Review by larshp — CHANGES_REQUESTED** [review]
Comments below

**Comment by larshp** [comment]
I dont think this should be a special case, it should be generic error handling from the create repo page.

1: User opens the repo page, enters data
2: Click create/clone

In my case the repository was created, but the UI stayed on the create repo page. If the repository is created, then the UI should show the repository, if not created then show error.

I have not looked into the details, but I guess there are more scenarios than the `.abapgit.xml` file issue

**Comment by mbtools** [comment]
maybe this check should be a sep method that's called from the UI only

```abap
find_remote_dot_abapgit

    ELSEIF lines( mt_remote ) > c_new_repo_size.
      " Less files means it's a new repo (with just readme and license, for example) which is ok
      zcx_abapgit_exception=>raise( |Cannot find .abapgit.xml - Is this an abapGit repo?| ).
```

**Comment by larshp** [comment]
hmm, yea, but still the repository is created when the exception occurs

I think,
a: move the error mentioned above, as it should just be a warning
b: there is something wrong with general error handling during creation and the UI flow, might be outside scope for this PR

**Comment by mbtools** [comment]
- Removed exception during `new_online`
- Added warning to repo view
image

File: src/zcl_abapgit_repo_content_list.clas.abap [hunk 1]
```diff
@@ -190,6 +194,27 @@ CLASS ZCL_ABAPGIT_REPO_CONTENT_LIST IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD check_for_dot_abapgit.
```
Comment (larshp) [inline src/zcl_abapgit_repo_content_list.clas.abap:197]: suggest renaming this method to "check_repo_size" ?

**Review by larshp — CHANGES_REQUESTED** [review]

**Review by larshp — APPROVED** [review]

**Pull request merged**